docs/development/app_development/pyfunc_components.rst

```diff
@@ -0,0 +1,24 @@
+.. _pyfunc_components:
+
+Pyfunc Components
+=================
+
+Pyfunc components are generalized python component that can be configured to behave as a custom python component entirely through component parameters and application arguments. A pyfunc component
```

Comment:
Add 's' to end of "generalized python component"?

Reply:
Thanks, I'll extend this further in collaboration with Andreas as some parameters like func_code need further attention